v2/api/containerservice/customizations/managed_clusters_agent_pool_extensions.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/pkg/errors"
 	"sigs.k8s.io/controller-runtime/pkg/conversion"
 
-	containerservice "github.com/Azure/azure-service-operator/v2/api/containerservice/v1api20230201/storage"
+	containerservice "github.com/Azure/azure-service-operator/v2/api/containerservice/v1api20230701/storage"
```

Comment:
Why not 2023-10-01, which is the latest GA API version?

Reply:
Updated to this version.